run.sh

```diff
@@ -265,6 +268,7 @@ if [ "${NETWORK}" == "regtest" ]; then
     else
         /srv/explorer/$DAEMON/bin/${DAEMON}d -conf=/data/.$DAEMON.conf -datadir=/data/$DAEMON -daemon
     fi
+    cli -rpcwait loadwallet default || cli createwallet default
```

Comment:
This has side effect of reporting an error

```
Elements Core starting
error code: -18
error message:
Wallet file verification failed. Failed to load database path '/data/liquid/liquidregtest/wallets/default'. Path does not exist.
```

Reply:
The error could be silenced with something like `cli -rpcwait loadwallet default 2>/dev/null || ...`

Reply:
the error is only in regtest anyway right? it can probably just be ignored

Reply:
Added echo of message in amended commit